tests/tests_pytorch/models/test_amp.py

```diff
@@ -24,47 +23,7 @@
 from pytorch_lightning import Trainer
 from pytorch_lightning.demos.boring_classes import BoringModel, RandomDataset
 from tests_pytorch.helpers.runif import RunIf
-
-
-class AMPTestModel(BoringModel):
```

Comment:
let's put it back if we don't need to move it? I think the other test you wrote is perfectly fine :)

Reply:
Makes sense, moved it back.